### ISSUE TITLE
Inline plotting returning error

### ISSUE THREAD

**Issue by medwatt** [luk400/vim-jukit#25]
When I execute `plt.show`, I get the error given below. If I disable inline image, the code runs fine. So, the issue seems to be with inline images.

```
TypeError                                 Traceback (most recent call last)
Input In [2], in <cell line: 2>()
      1 plt.plot(x, y, "r-", lw=2.0)
----> 2 plt.show()

File ~/.local/lib/python3.10/site-packages/matplotlib/pyplot.py:409, in show(*args, **kwargs)
    365 """
    366 Display all open figures.
    367 
   (...)
    406 explicitly there.
    407 """
    408 _warn_if_gui_out_of_main_thread()
--> 409 return _get_backend_mod().show(*args, **kwargs)

File ~/.local/share/nvim/site/pack/packer/start/vim-jukit/helpers/matplotlib-backend-kitty/__init__.py:97, in JukitBackend.show(cls, *_, **kwargs)
     95 scaling = kwargs.get("scaling", 0.9)
     96 align = kwargs.get("align", "center")
---> 97 manager.show(scaling, align)

TypeError: FigureManagerBase.show() takes 1 positional argument but 3 were given

```

**Comment by luk400** [comment]
I'll try to reproduce your issue tonight when I have time, please tell me the following specs:
* your OS
* kitty version
* ipython version
* matplotlib version
* Vim / Neovim version

**Comment by medwatt** [comment]
> I'll try to reproduce your issue tonight when I have time, please tell me the following specs:
> 
>     * your OS
> 
>     * kitty version
> 
>     * ipython version
> 
>     * matplotlib version
> 
>     * Vim / Neovim version

OS: Arch Linux

Kitty: 0.26.2

IPython: 8.4.0

matplotlib: 3.6.0

Neovim: 0.7.2

**Comment by luk400** [comment]
Alright so I was able to reproduce the problem, the new matplotlib release (3.6.0) is the the cause of the error, thanks for making me aware of this. For now, everything should work as expected if you just revert to e.g. version 3.5.3. I'll implement a fix to also support version 3.6 as soon as I have time, most likely sometime next week.

**Comment by luk400** [comment]
Problem should be fixed now if you reinstall the plugin, thanks for reporting this bug!
